buyer-api/src/recurrent/checkBatPayBalance.js

```diff
@@ -52,10 +62,15 @@ export const checkBatPayBalance = async () => {
  */
 export const sendDeposit = async (event) => {
   const { owner, spender, value: amount } = event;
-  const { address } = await getAccount();
-  if (address.toLowerCase() !== owner.toLowerCase()) return;
+  const account = await getAccount();
+  if (account.address.toLowerCase() !== owner.toLowerCase()) return;
   if (spender.toLowerCase() !== BatPay.options.address.toLowerCase()) return;
-  await addTransactionJob('Deposit', { amount });
+
+  const enoughBalance = await hasEnoughBatPayBalance(account);
+  if (!enoughBalance) {
```

Comment:
Since now we have super descriptive methods `hasEnoughBatPayBalance` and `hasBatPayEnoughTokenAllowance` (thank you for that 🙌 ), we could use them directly in the if conditions right? I think they are expressive enough to understand the if, and we avoid the unnecessary assignments of `enoughBalance` variables.

Reply:
You mean something like:
```suggestion
  if (await hasEnoughBatPayBalance(account)) {
    logger.info('BatPay Balance Check :: No deposit needed');
    return;
```

Reply:
Yes, on the other lines as well